src/com/linkedin/parseq/ParSeqGlobalConfiguration.java

```diff
@@ -0,0 +1,50 @@
+package com.linkedin.parseq;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+
+/**
+ * Global parseq configuration, applies to all Engine and Task instances
+ */
+public class ParSeqGlobalConfiguration {
+  private static final ParSeqGlobalConfiguration _instance = new ParSeqGlobalConfiguration();
+
+  private AtomicBoolean _crossThreadStackTracesEnabled = new AtomicBoolean(false);
+
+  public static ParSeqGlobalConfiguration getInstance() {
+    return _instance;
+  }
+
+  private ParSeqGlobalConfiguration() {
+  }
+
+  /**
+   * Returns current state of cross-thread (cross-task) stack tracing.
+   *
+   * Normally tasks are executed in different thread from the one creating it and at a different time. This makes it
+   * hard to debug because if task throws an exception, its call stack ends in the execution engine that actually
```

Comment:
I think it should be actually "its" here. If I replace with it's then the phrase "it is call stack ends" is not grammatically correct.